codesign/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
You don't need to add a new LICENSE file, this would be covered by the repo's existing license file: https://github.com/flutter/cocoon/blob/main/LICENSE

Reply:
sounds good! I have newer version that is almost ready for christopher review. Currently stuck up with flutter g3 roll but once that is in a good state i will push the new version up.